tests/testthat/test-model-compile.R

```diff
@@ -279,7 +279,8 @@ test_that("compile() works with pedantic=TRUE", {
   ")
   expect_message(
     mod_pedantic_warn <- cmdstan_model(stan_file, pedantic = TRUE),
-    "The parameter x was declared but was not used in the density calculation."
+    "The parameter x was declared but was not used",
```

Comment:
The text was a bit modified and formatted differently and this test works for 2.28 and earlier and 2.29+.